Formula/qpid-proton.rb

```diff
@@ -0,0 +1,38 @@
+class QpidProton < Formula
+  desc "High-performance, lightweight AMQP 1.0 messaging library"
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www.apache.org/dyn/closer.lua?path=qpid/proton/0.18.0/qpid-proton-0.18.0.tar.gz"
+  sha256 "7f54c3555a8482d439759c087d656d881369252af23f209f1fd5a7e4a8c59bd7"
+  head "git://git.apache.org/qpid-proton.git"
+
+  depends_on "cmake" => :build
+  depends_on "libuv"
+  depends_on "openssl"
+
+  def install
+    ENV["OPENSSL_ROOT_DIR"] = Formula["openssl"].opt_prefix
+
+    system "cmake", ".", "-DBUILD_BINDINGS=",
+                         "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=#{lib}",
+                         *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
```

Comment:
@ilovezfs This needs changing to `<<~EOS` as well, if you're making changes so this is good to merge 🙈.

Reply:
pig vomit

Reply:
🤣 This could probably be audited at this point, to be honest.

Reply:
Could but they're not actually equivalent :(

Reply:
Yeah, it's not a perfect match. I guess enough of them are being caught as-is without the audit.

Reply:
or we could adopt a silly convention like using a different heredoc delimiter for any such cases as a poor man's whitelist